yeat/cli/cli.py

```diff
@@ -57,13 +68,6 @@ def options(parser):
         default=".",
         help="output directory; default is current working directory",
     )
-    parser.add_argument(
-        "--sample",
-        type=str,
-        metavar="S",
-        default="sample",
-        help="specify a unique sample name S for storing assembly results in the working directory; by default S=sample",
-    )
```

Comment:
Removed this flag. We will now use the `labels` of each algorithm instead.

For example, "canu" requires the user to provide `-p`, the assembly prefix. Instead of using the original user's input `--sample`, we can use the algorithm's `label` in the config file.